app/models/client.rb

```diff
@@ -321,6 +321,52 @@ def target_id=(value)
     Doi.transfer(client_id: symbol.downcase, target_id: target.id)
   end
 
+  def transfer(options = {})
+
+    if options[:target_id].blank?
+      Rails.logger.error "[Transfer] No target provider provided."
+      return nil
+    end
+
+    target_provider = Provider.where(symbol: options[:target_id]).first
+
+    if target_provider.blank?
+      Rails.logger.error "[Transfer] Provider doesn't exist."
+      return nil
+    end
+    
+    if target_provider.member_type == "consortium"
```

Comment:
direct_member can also not be target. Maybe better to do the other way around, i.e. member_types that are allowed (direct_member and consortium_organization).